pages/national_resources/no_resources.md

```diff
@@ -147,7 +147,15 @@ national_resources:
   - name: openscience.no
     description: general information about open research, especially open access and guidance for researchers and institutions
     url: https://www.openscience.no/
-    
+  - name: Nettskjema
+    description: Nettskjema is a solution for designing and managing data collections using online forms and surveys.
```

Comment:
```suggestion
    description: Nettskjema is a solution for designing and managing data collections using online forms and surveys.  It can be used for collecting sensitive data and offers a high degree of security and privacy.
```